RealEstate/apps/core/templates/core/houseEval.html

```diff
@@ -5,96 +5,120 @@
-						{% if category.description %}
-							<p>
-								{{ category.description }}
-							</p>
-						{% endif %}
-					</div>
-					<div class="col-sm-6 vcenter rating">
-						<div class="inner-rating" >
-							<center>
-                                <span class="help-text" style="padding-right: 15px">{{min_choice}}</span>
-                                <input id="category_{{category.id}}" class='slide' data-slider-id='slide' type="text" data-slider-min={{min_score}} data-slider-max={{max_score}} data-slider-step="1" data-slider-value="{% if not score %}{{default_score}}{% else %}{{ score }}{% endif %}"/>
-                                <span class="help-text" style="padding-left: 15px">{{max_choice}}</span>
-							</center>
-						</div>
+</div>
+<p id="save-status" class="pull-right"></p>
+<ul class="list-group">
+	{% for category, score in grades %}
+		<li class="list-group-item">
+			<div class="row">
+				<div class="col-sm-6 vcenter">
+					<h4>{{category.summary}}
+						<span id='update_{{category.id}}' class='glyphicon glyphicon-ok icon-ok'></span>
+					</h4>
+					{% if category.description %}
+						<p>
+							{{ category.description }}
+						</p>
+					{% endif %}
+				</div>
+				<div class="col-sm-6 vcenter rating">
+					<div class="inner-rating" >
+						<center>
+							<span class="help-text" style="padding-right: 15px">{{min_choice}}</span>
+							<input id="category_{{category.id}}" class='slide' data-slider-id='slide' type="text" data-slider-min={{min_score}} data-slider-max={{max_score}} data-slider-step="1" data-slider-value="{% if not score %}{{default_score}}{% else %}{{ score }}{% endif %}"/>
+							<span class="help-text" style="padding-left: 15px">{{max_choice}}</span>
+						</center>
 					</div>
 				</div>
-			</li>
-		{% endfor %}
-	</ul>
-	<form style="margin-bottom:20px" action="" role="form" method="post">
-		{% bootstrap_form form%}
-		{% csrf_token %}
-		<div style="margin-top: 40px">
-			<center>
-				{% buttons submit='Save Evaluation' %}{% endbuttons %}
-				{% bootstrap_messages %}
-
-			</center>
-		</div>
-	</form>
+			</div>
+		</li>
+	{% endfor %}
+</ul>
 {% endblock %}
 
 {% block scripts %}
 <script src="{% static "slide.js" %}"></script>
 <script>
-    var jsScores = {{js_scores|safe}};
-	$(".slide").slider({
-		formatter: function(value) {
-            return jsScores[value] || "";
+var jsScores = {{js_scores|safe}}
+$(".slide").slider({
+	formatter: function(value) {
+          return jsScores[value] || "";
+	}
+});
+$(".slide").on("change", function(slideEvt) {
+	var timer = setTimeout(showStatus, 1500);
+	clearTimeout(timer);
+	function showStatus() {
+		$("#update_"+categoryID).stop().show();
+		$("#update_"+categoryID).fadeOut(2000);
+		$("#save-status").text("All changes saved.").show().css("color", "#333");
+	}
+	var categoryID = $(this).attr("id").toString().replace(/category_/i, "");
+	$("#save-status").text("saving...").css("color", "#888");
+	$.ajax({
```

Comment:
The ajax request should be debounced so our server doesnt explode.  See [this](http://benalman.com/code/projects/jquery-throttle-debounce/examples/debounce/) for an example of the difference in network traffic.